### DISCUSSION TITLE
How can I generate query and resolver automatic in graphql playground when I add a new model

### DISCUSSION THREAD

**Discussion by microwin168** [prisma/prisma#6888] [Q&A] [answered]
How can I generate query and resolver automatic in graphql playground when I add a new model?
Is it need some configs or packges?

**Comment by 2color** [comment, marked as answer]
Hey @microwin168 👋,

If I understand you correctly, you are trying to build a GraphQL API with your Prisma project.

While Prisma helps you handle database operations in a GraphQL API, it isn't a GraphQL solution per se.

I would highly recommend checking out the [following docs page](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql) about how Prisma fits into GraphQL.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
